CoreDistributedCache/NHibernate.Caches.CoreDistributedCache.Redis/NHibernate.Caches.CoreDistributedCache.Redis.csproj

```diff
@@ -10,7 +10,11 @@
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\NHibernate.Caches.snk</AssemblyOriginatorKeyFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageReleaseNotes>* Improvement
+    <PackageReleaseNotes>## Possible breaking change
+    * The obsolete Microsoft.Extensions.Caching.Redis dependency has been changed for Microsoft.Extensions.Caching.StackExchangeRedis
```

Comment:
Thinking about it again, that is a binary breaking change... So it should not be there, and it should wait for a 6.0.
But since I would rather not publish a 6.0 depending on a 5.x NH, it could then wait quite long.
And for most cases, since caches are injected through interfaces not bound to that dependency, the change will actually not be breaking. So I am for letting it go into 5.6 rather than reverting it.

Reply:
Well is-it really binary breaking? No interface at all was changed, only reflection/hard casting may be broken.

Reply:
Technically it is a breaking change as `Microsoft.Extensions.Caching.StackExchangeRedis` references `StackExchange.Redis` version `2.0.593` which has [some binary breaking](https://stackexchange.github.io/StackExchange.Redis/ReleaseNotes#20495) for profiling API from version `1.2.6` used in `Microsoft.Extensions.Caching.Redis`. As most users do not use the profiling API I would also letting it go into `5.6` as NH 6.0 will probably not be released anytime soon.